README.md

```diff
@@ -1 +1,208 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  
+
+  <h3><b>Benjamin's README</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [webpack_To_Do_List] <a name="about-project"></a>
+
+
+
+**[webpack_To_Do_List]** is a Setup of the project repository
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+
+<details>
+  
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+    <li><a href="https://www.w3schools.com/css/">Javascript</a></li>
+    <li><a href="https://www.w3schools.com/css/">Webpack</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+
+<ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+    <li>Webpack</li>
+  </ul>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+<ul>
+    <li><a href="#">Coming soon.......</a></li>
+    
+  </ul>
+  
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+
+Clone the code from project using terminal into your local machine and then install node modules using npm install on windows or sudo npm install on mac.
+
+
+
+### Setup
+
+
+>Clone this repository to your desired folder using:
+
+ <h1>Setup SSH-key and use this command</h1>
+ git clone https://github.com/BenjaminMatembe/To_Do_List.git 
+
+ <h1>Using HTTPS</h1>
+
+ git clone git@github.com:BenjaminMatembe/To_Do_List.git
+
+
+
+
+### Install
+
+Install node_modules dependencies by:
+
+>running npm install or yarn install
+
+### Usage
+
+To run the project, execute the following command:
+
+>right click on index.html and open with live server
+
+### Run tests
+
+To run tests, run the following command:
+
+>coming soon
+
+### Deployment
+
+This project will be deployed using github pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 **Benjamin**
+
+- GitHub: [@githubhandle](https://github.com/BenjaminMatembe)
```

Comment:
[OPTIONAL]
- [ ] Kindly add your Linkedin and Twitter account to the README file. That will make easier to visitors or recruiters to reach out to you.